src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/waypoints_pending_resumption_handler.h

```diff
@@ -78,15 +78,22 @@ class WayPointsPendingResumptionHandler
                                    const int32_t corr_id);
   smart_objects::SmartObjectSPtr CreateSubscriptionRequest();
 
-  struct ResumptionAwaitingHandling {
-    const uint32_t app_id;
-    WayPointsAppExtension& ext;
-    resumption::ResumptionRequest request_to_send_;
+  void ProcessNextPendingResumption(const smart_objects::SmartObject& response);
```

Comment:
Add doxygen for new functions

Reply:
It is totally unclear that is the reason to pass `response` as a parameter for NextPendingResumption. Need to be described properly or reworked

Reply:
Because we need to know whether response was successful for previous app. Yes, we can use `application_manager_->IsAnyAppSubscribedForWayPoints()` instead of` resumption::IsResponseSuccessful(response)`, but in this case we still need response for `RaiseFakeSuccessfulResponse` method, where we change corr_id from this response to corr_id from initial request, saved for this app.

Reply:
Please use `IsAnyAppSubscribedForWayPoints` if it is possible. 
Creating new fake response for fake requests is better then Reusing existing response and replacing corr_id parameter

Reply:
And still don't see doxygens

Reply:
Suggested changes are implemented in ff6fc20e24f1ce4bcff713dc585f904332c9a812